src/App.js

```diff
@@ -3,60 +3,108 @@ import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
 
-const App = () => {
-  const [todoList, setTodoList] = React.useState([]);
-  const [isLoading, setIsLoading] = React.useState(true);
 
+const AIRTABLE_API_URL = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
```

Comment:
Great use of a global variable here!

Reply:
It was a suggestion from one of our mentors and I really liked it. I try to work on not repeating myself unnecessary but it's not always easy.